first searchbar commit

Barosz30 opened the pull request:
- [DEMO LINK](https://Barosz30.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Barosz30.github.io/layout_search-bar-airbnb/report/html_report/)

mgrinko committed:
Merge pull request #3662 from tiserett/change-checklist

JoyCoffeeAddict commented on src/style.css:
```diff
@@ -0,0 +1,101 @@
+  width: 100%;
+  border: none;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  padding-left: 18px;
+
+  &:focus {
+    text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+    color: #3d4e61;
+    font-family: "Avenir", sans-serif;
+    font-size: 1rem;
```
Decide if you want to use `rem` for font-size or `px`. Usually it's `rem` for font-sizes

JoyCoffeeAddict commented on src/style.css:
```diff
@@ -0,0 +1,101 @@
+  &:focus-within {
+    border: none;
+    text-shadow: #3d4e6155 1px 3px 3px;
+    background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
This value is repeated, put it in a variable

JoyCoffeeAddict commented on src/style.css:
```diff
@@ -0,0 +1,101 @@
+    border: none;
+    text-shadow: #3d4e6155 1px 3px 3px;
+    background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+    color: #3d4e61;
```
The same with this value.

JoyCoffeeAddict reviewed (changes requested):
Small issues, other than that, good job

choeqq reviewed (approved):
Hi, my name is Kacper, I am a mentor in other group, and I'll be helping mentors team with checking homework, until they'll catch up with your group's work 😄 

The solution looks good to me ✅

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #1 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
